pkg/k8shandler/status.go

```diff
@@ -257,14 +256,14 @@ func updateNodeConditions(clusterName, namespace string, status *api.Elasticsear
 			}
 
 			if !thresholdEnabled {
-				// disk threshold is not enabled, just return
-				return
+				// disk threshold is not enabled, continue to next node
+				continue
 			}
 
 			usage, percent, err := GetNodeDiskUsage(clusterName, namespace, nodeName)
 			if err != nil {
-				logrus.Debugf("Unable to get disk usage for %v", nodeName)
-				return
+				logrus.Warnf("Unable to get disk usage for %v", nodeName)
```

Comment:
this could fail for a couple reasons like the cluster not being ready yet. I don't think we want to print a `Warn` for each of those times.